prow/config/jobs/istio.yaml

```
@@ -78,6 +78,12 @@ jobs:
     requirements: [kind]
     modifiers: [optional]
 
+  - name: integ-istioio-k8s-tests
+    type: presubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, make test.integration.istioio.kube.presubmit]
```

```suggestion
    command: [entrypoint, prow/integ-suite-kind.sh, test.integration.istioio.kube.presubmit]
```

I don't think this will work as it is